roles/ansible-runner/tasks/main.yml

```diff
@@ -83,3 +70,6 @@
 
 - include: runner.yml
   with_items: "{{ ansible_runner_tasks }}"
+
+- when: ara_enabled | default(False) | bool
+  include: ara.yml
```

Comment:
Can you flip these? Do the include, with the conditional after? Makes more sense as the conditional is applied to every task within, rather than making the inclusion itself conditional.